scripts/changeScanner/index.js

```diff
@@ -14,28 +14,24 @@ const requiredChanges = ['CHANGELOG.md', packageFileName];
 
 const changedPackages = getChanges();
 
-const getChangedFilePath = ({ packageName, matchFile }) =>
-  changedPackages[packageName].find(changedFile =>
-    changedFile.includes(matchFile),
-  );
+const getChangedFilePath = ({ packageName, matchFile }) => {
+  const packageChanges = changedPackages[packageName];
+
+  return packageChanges
+    ? packageChanges.find(changedFile => changedFile.includes(matchFile))
+    : '';
+};
 
 const isMissingRequiredChangedFile = ({ packageName, requiredFile }) =>
   !changedPackages[packageName].some(changedFile =>
     changedFile.includes(requiredFile),
   );
 
-const someDepsHaveChanged = ({ localPackageFile, remotePackageFile }) => {
-  const { dependencies: localDeps, devDependencies: localDevDeps } = JSON.parse(
-    localPackageFile,
-  );
-  const {
-    dependencies: remoteDeps,
-    devDependencies: remoteDevDeps,
-  } = JSON.parse(remotePackageFile);
-  const depsHaveChanged = !equals(localDeps, remoteDeps);
-  const devDepsHaveChanged = !equals(localDevDeps, remoteDevDeps);
```

Comment:
Not sure I follow the changes in this file; why are devDeps not no longer checked?

Reply:
They don't affect the root yarn.lock